modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/configurator/ClientExtensionProjectConfigurator.java

```diff
@@ -552,6 +614,29 @@ private void _configureConfigurationDefault(Project project) {
 		defaultConfiguration.extendsFrom(archivesConfiguration);
 	}
 
+	private void _configureDeployProfileTask(
+		Project project, File clientExtensionYamlFile,
+		File overrideClientExtensionYamlFile, String profileName) {
+
+		String taskName = "deploy" + StringUtil.capitalize(profileName);
```

Comment:
I'm liking what I'm seeing, but wondering if `profileName` is the correct name.  deployMode? or just  thinking... i don't think i have a better name at the moment.

Reply:
My one thought is that this concept _might_ go beyond just deploy at some point. `profile` seems applicable but generic enough to work for other stuff. 

Maybe we ask - what would we call `client-extension.yaml`? A configuration? A mapping? A definition?